datatable/types.py

```diff
@@ -231,6 +246,16 @@ def ___new___(cls, value):
     stype.obj64: ctypes.py_object,
 }
 
+_stype_extrema = {
+    stype.bool8: (0, 1),
+    stype.int8: (-255, 255),
+    stype.int16: (-65536, 65536),
+    stype.int32: (-4294967296, 4294967296),
+    stype.int64: (-18446744073709551616, 18446744073709551616),
+    stype.float32: (unpack(">f", b'\xff\x7f\xff\xff')[0], unpack(">f", b'\x7f\x7f\xff\xff')[0]),
```

Comment:
```suggestion
    stype.float32: (float.fromhex("-0x1.fffffep+127"), float.fromhex("0x1.fffffep+127")),
```
this feels cleaner, and also avoids loading an extra module